test/run_spec.js

```diff
@@ -0,0 +1,99 @@
+const { spawn } = require("child_process");
+const fs = require("fs");
+const os = require("os");
+const path = require("path");
+const prettier = require("prettier");
+const prettierRuby = require("../src/ruby");
+
+const expectedMinitestFiles = [
+  "alias.rb",
+  "array.rb",
+  "binary.rb",
+  "next.rb",
+  "numbers.rb",
+  "kwargs.rb",
+  "regexp.rb"
+];
```

Comment:
This should probably be colocated with the new `*.spec.js` files as part of a passed in parameter - but I want to avoid a merge conflict with the open PRs for now. Hopefully it's a short lived code smell either way though, until all the ruby files become minitests :+1: